tests/ia/pages/api/alertsAPI.js

```diff
@@ -1,5 +1,6 @@
 const { I } = inject();
 const assert = require('assert');
+const { api } = require('@pagerduty/pdjs');
```

Comment:
the naming API? confuses me

Reply:
this is the name of an object from the library. This is how they call it. https://github.com/PagerDuty/pdjs#rest-api